src/spec/items.spec.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../app');
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('items', () => {
+  it('should return all items', async () => {
+    const res = await chai.request(server)
+      .get('/items');
+    res.should.have.status(200);
+    res.body.should.be.an('array');
+    res.body.length.should.be.greaterThan(0);
```

Comment:
Well, the problem there is that it will literally break every update if we do an equal, since there's always more objects. If we want to set a known minimum, that's fine by me, I just don't want to tightly couple it too much.

If we want structure checks, I'd ideally like to check that in the other similar routes, since those have a consistent structure.

Reply:
I guess i didn't explain how to equal would work, i was thinking of getting the items like new Items().length but i don't know if translations would interfere with it.

By structure checks you mean the structure of the returned data? If so i don't think it's a good idea, it would add too much static data to the tests.

Reply:
> I guess i didn't explain how the equal would work, i was thinking of getting the items like new Items().length but i don't know if translations would interfere with it.
> 

Ohhh, that should be fine

> By structure checks you mean the structure of the returned data? If so i don't think it's a good idea, it would add too much static data to the tests.

Maybe just to structure tests on specific items with specific check, like name and such, no need to have full coverage of  every object, basically just making sure that critical data didn't disappear

Reply:
> Maybe just to structure tests on specific items with specific check, like name and such, no need to have full coverage of every object, basically just making sure that critical data didn't disappear

It's a great idea, in items we could check for `tradable, name, uniqueName and type`.
How should we approach this in tests? A single test for required structure or different tests for each of the required key / values?

Reply:
Single test should be plenty, since it's the same object, just 4 fields